terraform/modules/security_group/main.tf

```diff
@@ -0,0 +1,98 @@
+resource "aws_security_group" "sg" {
+    name_prefix = "${var.name}-${var.vpc_id}-"
+    description = " For ${var.name} in VPC ${var.vpc_id}"
+    vpc_id = "${var.vpc_id}"
+    tags {
+        Name = "${var.name}-${var.vpc_id}"
+    }
+}
+
+resource "aws_security_group_rule" "all_internal_ingress" {
```

Comment:
Is there a way we can avoid allowing everything as ingress?
(Not a priority, and not required for initial dpeloyment.)